Tella/Supporting Files/fr.lproj/Localizable.strings

```diff
@@ -206,5 +213,42 @@ If you want to take a picture or a video, proceed in the following way:\
 "Settings_SettAbout_ContactUs" = "Nous contacter";
 "Settings_SettAbout_PrivacyPolicy" = "Politique de confidentialité";
 
+"Reports_AppBar" = "Reports";
+"Reports_AppBar_SubHead" = "Report";
```

Comment:
@ValbuenaG I think Report is not subhead it is an AppBar, it's better to rename it Reports_Report_AppBar

Reply:
thanks! just committed the change!